pkg/scheduler/framework/plugins/volumebinding/binder.go

```diff
@@ -65,7 +65,7 @@ const (
 	// ErrReasonBindConflict is used for VolumeBindingNoMatch predicate error.
 	ErrReasonBindConflict ConflictReason = "node(s) didn't find available persistent volumes to bind"
 	// ErrReasonNodeConflict is used for VolumeNodeAffinityConflict predicate error.
-	ErrReasonNodeConflict ConflictReason = "node(s) had volume node affinity conflict"
+	ErrReasonNodeConflict ConflictReason = "node(s) didn't match PersistentVolume's node affinity/selector"
```

Comment:
I think this is exactly node affinity conflict, https://github.com/kubernetes/kubernetes/blob/a18b4a8d97151a25c4d8088839d77e6ba482cac3/staging/src/k8s.io/api/core/v1/types.go#L433

Any thoughts?

Reply:
I agree, this is a node affinity conflict, but I do not think it is easy for average user to understand where this affinity conflict happened from the old message. My proposed message shows that there is a affinity conflict and it is because node and PV's node have affinity conflict, and it points them in the right direction during debugging.

Reply:
I agree with @rhrmo. Users often don't know that there even *is* something like PersistentVolume nodeAffinity and they look for it in Pod spec. The message should point them to PVs used by the Pod and users can find nodeAffinity there easily.

Reply:
> there even is something like PersistentVolume nodeAffinity and they look for it in Pod spec

Make sense to me. Can we clear the messages then, the `node affinity` is somehow the superset of `selector` from the go structure definition, let's make it like, wdyt?

```
node(s) didn't match PersistentVolume's node affinity
```

Reply:
sure, sounds good to me, I will update it